### PR TITLE
Note EF Core probe package requirement

### LINKED ISSUE #9983: Required package for AddDbContextCheck
The EF Core DbContext Probe section should specify that it requires the `Microsoft.Extensions.Diagnostics.HealthChecks.EntityFrameworkCore` package

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: c0cbb094-29e9-740a-9fd6-db372641d753
* Version Independent ID: 52d8efc9-a336-71d9-6d90-98173cbf6bf4
* Content: [Health checks in ASP.NET Core](https://docs.microsoft.com/en-us/aspnet/core/host-and-deploy/health-checks?view=aspnetcore-2.2#health-check-publisher)
* Content Source: [aspnetcore/host-and-deploy/health-checks.md](https://github.com/aspnet/Docs/blob/master/aspnetcore/host-and-deploy/health-checks.md)
* Product: **aspnet-core**
* GitHub Login: @guardrex
* Microsoft Alias: **riande**

### PR DESCRIPTION
**Pull request by guardrex**
Fixes #9983 

Thanks @connorLaderer! :rocket:

### CODE REVIEW

File: aspnetcore/host-and-deploy/health-checks.md [hunk 1]
```diff
@@ -308,9 +310,17 @@ dotnet run --scenario db
...
 
-`AddDbContextCheck<TContext>` registers a health check for a `DbContext` (`TContext`). By default, the name of the health check is the name of the `TContext` type. An overload is available to configure the failure status, tags, and a custom test query.
+* Use [Entity Framework (EF) Core](/ef/core/).
+* Include a package reference to [Microsoft.Extensions.Diagnostics.HealthChecks.EntityFrameworkCore](https://www.nuget.org/packages/Microsoft.Extensions.Diagnostics.HealthChecks.EntityFrameworkCore/).
+
+`AddDbContextCheck<TContext>` registers a health check for a `DbContext` (`TContext`). An overload is available to configure the failure status, tags, and a custom test query.
+
+By default:
+
+* The `DbContextHealthCheck` calls EF Core's `CanConnectAsync` method. You can customize what operation is run when checking health using overloads of the `AddDbContextCheck` method.
```
Comment (scottaddie) [inline aspnetcore/host-and-deploy/health-checks.md]:
```suggestion
* The `DbContextHealthCheck` calls EF Core's `CanConnectAsync` method. You can customize what operation is run when checking health using `AddDbContextCheck` method overloads.
```

File: aspnetcore/host-and-deploy/health-checks.md [hunk 2]
```diff
@@ -308,9 +310,17 @@ dotnet run --scenario db
...
+The `DbContext` check confirms that the app can communicate with the database configured for an EF Core `DbContext`. The `DbContext` check is supported in apps that:
 
-`AddDbContextCheck<TContext>` registers a health check for a `DbContext` (`TContext`). By default, the name of the health check is the name of the `TContext` type. An overload is available to configure the failure status, tags, and a custom test query.
+* Use [Entity Framework (EF) Core](/ef/core/).
+* Include a package reference to [Microsoft.Extensions.Diagnostics.HealthChecks.EntityFrameworkCore](https://www.nuget.org/packages/Microsoft.Extensions.Diagnostics.HealthChecks.EntityFrameworkCore/).
+
+`AddDbContextCheck<TContext>` registers a health check for a `DbContext` (`TContext`). An overload is available to configure the failure status, tags, and a custom test query.
```
[the remark this answers is not in the document]
Comment (guardrex) [reply aspnetcore/host-and-deploy/health-checks.md]: It's the other way around 🙃 ... I'll fix it on the next commit.

File: aspnetcore/host-and-deploy/health-checks.md [hunk 3]
```diff
@@ -30,10 +30,12 @@ Health checks are usually used with an external monitoring service or container
 
 Reference the [Microsoft.AspNetCore.App metapackage](xref:fundamentals/metapackage-app) or add a package reference to the [Microsoft.AspNetCore.Diagnostics.HealthChecks](https://www.nuget.org/packages/Microsoft.AspNetCore.Diagnostics.HealthChecks) package.
 
-The sample app provides start-up code to demonstrate health checks for several scenarios. The [database probe](#database-probe) scenario probes the health of a database connection using [BeatPulse](https://github.com/Xabaril/BeatPulse). The [DbContext probe](#entity-framework-core-dbcontext-probe) scenario probes a database using an EF Core `DbContext`. To explore the database scenarios using the sample app:
+The sample app provides start-up code to demonstrate health checks for several scenarios. The [database probe](#database-probe) scenario probes the health of a database connection using [BeatPulse](https://github.com/Xabaril/BeatPulse). The [DbContext probe](#entity-framework-core-dbcontext-probe) scenario probes a database using an EF Core `DbContext`. To explore the database scenarios, the sample app:
```
Comment (scottaddie) [inline aspnetcore/host-and-deploy/health-checks.md]:
```suggestion
The sample app provides startup code to demonstrate health checks for several scenarios. The [database probe](#database-probe) scenario examines the health of a database connection using [BeatPulse](https://github.com/Xabaril/BeatPulse). The [DbContext probe](#entity-framework-core-dbcontext-probe) scenario examines a database using an EF Core `DbContext`. To explore the database scenarios, the sample app:
```

**Comment by guardrex** [comment]
I don't think "probes" was a problem. To avoid a repeated word, "checks" is better than "examines" imo.

**Pull request merged**